### PR TITLE
[ADD] States Complete

### PR DESCRIPTION
**Pull request by eilst**

### CODE REVIEW

File: project_event/models/project_project.py [hunk 1]
```diff
@@ -57,20 +57,32 @@ def _compute_event_log_count(self):
 
     @api.multi
     def action_cancel(self):
+        if self.task_state == 'accepted':
```
Comment (rimbendhaou) [inline project_event/models/project_project.py:60]: there is no cancel of the calendar.event to cancel resource reservation?

File: project_event/models/project_project.py [hunk 2]
```diff
@@ -57,20 +57,32 @@ def _compute_event_log_count(self):
...
         self.write({'state': 'option'})
 
     @api.multi
     def action_draft(self):
         self.write({'state': 'draft'})
 
+    @api.multi
+    def action_postpone(self):
+        if self.task_state == 'accepted':
+            self.send_message('postponed')
```
Comment (rimbendhaou) [inline project_event/models/project_project.py:89]: what about resource reservation?

File: project_event/models/project_project.py (hunk 1 above)
Comment (rimbendhaou) [reply project_event/models/project_project.py:60]: if an event is canceled its activities and tasks must also be canceled.
Same thing for activity (if it is canceled, its tasks should be canceled)

**Review by rimbendhaou — CHANGES_REQUESTED** [review]

File: project_event/models/project_project.py (hunk 1 above)
Comment (eilst) [reply project_event/models/project_project.py:60]: Yes I m adding this functionality in a moment

Comment (eilst) [reply project_event/models/project_project.py:60]: Done

File: project_event/models/project_project.py (hunk 2 above)
Comment (eilst) [reply project_event/models/project_project.py:89]: What?

Comment (eilst) [reply project_event/models/project_project.py:89]: Ok, I get it. Done

**Review by rimbendhaou — APPROVED** [review]

**Pull request merged**